sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,25 +1,45 @@
 # Release History
 
-## 1.8.0-beta.4 (Unreleased)
+## 1.8.0 (2023-02-02)
 
 ### Features Added
 
+- Added support for parsing space character in place of 'T' in RFC3339 DateTimes.
+- Added support for HTTP proxy servers, both unauthenticated and with Plain authentication.
+- Added universal support for several TLS options:
+  - Added the ability to set the expected TLS root certificate for TLS connection (useful if a proxy server uses a TLS certificate that is not chained to a known root).
+  - Added the ability to enable TLS certificate revocation list checks (off by default).
+    - For libcurl only: Allow TLS connection to succeed if CRL retrieval fails.
+    - *NOTE*: This change only applies if libcurl is built using the OpenSSL crypto backend. It does NOT apply if libcurl uses the schannel (Windows default) or SecureTransport (macOS/iOS default).
+
 ### Breaking Changes
 
+- Bearer token authentication will not work for endpoint URL protocol schemes other than `"https"`. This ensures token security and is consistent with the Azure SDKs for other languages.
+- Removed `noexcept` specification from `Azure::DateTime::clock::now()`.
+
+- Updated retry policy timeouts to conform to Azure guidelines.
+  - The default delay between retries is changed from 4 seconds to 800ms.
+  - The maximum retry delay is changed from 2 minutes to 60 seconds (one minute).
+
+  If the original behavior is desired, customers can adjust these timeouts by changing the `RetryDelay` and `MaxRetryDelay` fields in the `RetryOptions` structure.
+
 ### Bugs Fixed
 
-- Fixed bug in WinHTTP client which caused the `IgnoreUnknownCertificateAuthority` and `EnableCertificateRevocationListCheck` fields to be
-ignored if they were passed in from `TransportOptions`.
+- Fixed bug in WinHTTP client which caused the `IgnoreUnknownCertificateAuthority` and `EnableCertificateRevocationListCheck` fields to be ignored if they were passed in from `TransportOptions`.
 - [[#4206]](https://github.com/Azure/azure-sdk-for-cpp/issues/4206) Fixed connectivity issues which can occur if a TCP connection is dropped prematurely. (A community contribution, courtesy of _[ahojnnes](https://github.com/ahojnnes)_)
 
+### Other Changes
+
+- Added the ability to consume version 1.1.1n of OpenSSL.
+- Added support for Identity token caching, and for configuring token refresh offset in `BearerTokenAuthenticationPolicy`.
+- Improved cancellation support for WinHTTP transport.
```

Comment:
```suggestion
- Improved cancellation support for WinHTTP transport.
- Internal changes to support OpenTelemetry 1.17.0.
```

Reply:
Where do we reflect changes to the API surface of Azure Core used by service authors?